src/components/DelayRender.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class DelayedRender extends React.Component {
```

Comment:
Is it better to match the class name `DelayedRender` and the file name `DelayRender.js`?

Reply:
good catch